addon/components/frost-list-toggle-item.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember'
+const {Component, ViewUtils} = Ember
+import computed from 'ember-computed-decorators'
+import layout from '../templates/frost-list-toggle-item'
+
+export default Component.extend({
```

Comment:
As discussed, this can be removed in favor of just a frost-button (or a styled div with an action on it)